dev/com.ibm.websphere.appserver.api.sipServletSecurity.1.0/bnd.bnd

```diff
@@ -0,0 +1,28 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion: 1.0
+
+Bundle-Name: WebSphere SIP Container security API
+Bundle-Description: WebSphere SIP Container security Support, version ${bVersion}
+Bundle-SymbolicName: com.ibm.websphere.appserver.api.sipServletSecurity.1.0
+
+Import-Package: com.ibm.websphere.security.tai.extension
+
+Export-Package: com.ibm.websphere.security.tai.extension
```

Comment:
The com.ibm.wsspi.security.tai.extension also needs to be exported.

Reply:
done